docs/source/howto/share_data.rst

```diff
@@ -0,0 +1,248 @@
+
+During import, AiiDA will avoid identifier collisions and node duplication based on UUIDs (and email comparisons for ``User`` entries).
+By default, existing entities will be updated with the most recent changes.
+Node extras and comments have special modes for determining how to import them - for more details, see ``verdi import -h``.
+
+.. tip:: The AiiDA archive format has evolved over time, but you can still import archives created with previous AiiDA versions.
+    If an outdated archive version is detected during import, you will be prompted to confirm automatic migration of the archive file.
+
+    You can also use ``verdi export migrate`` to update existing archive files once and for all.
+
+.. tip:: In order to get a quick overview of an archive file *without* importing it into your AiiDA profile, use ``verdi export inspect``:
+
+    .. code-block:: console
+
+        $ verdi export inspect sssp-efficiency.aiida
+        --------------  -----
+        Version aiida   1.2.1
+        Version format  0.9
+        Computers       0
+        Groups          0
+        Links           0
+        Nodes           85
+        Users           1
+        --------------  -----
+
+    Note: For archive versions 0.2 and below, the overview may be inaccurate.
+
+
+.. _how-to:data:serve:
+
+Serving data through the REST API
+=================================
+
+The AiiDA REST API allows to query your AiiDA database over HTTP(S).
+
+.. note::
+
+    As of October 2020, the AiiDA REST API does *not* yet support workflow management.
+    This feature is, however, part of the `AiiDA roadmap <https://github.com/aiidateam/aiida-core/wiki/AiiDA-release-roadmap>`_.
+
+Launching the REST API
+^^^^^^^^^^^^^^^^^^^^^^
+
+Start serving data from your default AiiDA profile via the REST API:
+
+.. code-block:: console
+
+    $ verdi restapi
+     * REST API running on http://127.0.0.1:5000/api/v4
+     * Serving Flask app "aiida.restapi.run_api" (lazy loading)
+     * Environment: production
+       WARNING: This is a development server. Do not use it in a production deployment.
+       Use a production WSGI server instead.
+     * Debug mode: off
+     * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
+
+The REST API is now running on port ``5000`` of your local computer.
+
+Like all ``verdi`` commands, you can select a different AiiDA profile via the ``-p PROFILE`` option:
+
+.. code-block:: bash
+
+    verdi -p <another_profile> restapi
+
+
+.. note::
+
+    REST API version history:
+
+     * ``aiida-core`` >= 1.0.0b6: ``v4``
+     * ``aiida-core`` >= 1.0.0b3, <1.0.0b6: ``v3``
+     * ``aiida-core`` <1.0.0b3: ``v2``
+
+
+Querying the REST API
+^^^^^^^^^^^^^^^^^^^^^
+
+
+Open the base URL of the REST API (by default: `http://127.0.0.1:5000/api/v4 <https://aiida-dev.materialscloud.org/2dstructures/api/v4/nodes>`_) in your web browser in order to get an overview of the available endpoints.
```

Comment:
Why mislead here and point to a MaterialsCloud REST API if you're showing the URL of something different?
Perhaps make it more clear that you're sending people to MaterialsCloud
```suggestion
Open the base URL of the REST API (by default: http://127.0.0.1:5000/api/v4, `example on MaterialsCloud <https://aiida-dev.materialscloud.org/2dstructures/api/v4/nodes>`_) in your web browser in order to get an overview of the available endpoints.
```

Reply:
This was a copy/paste error, will fix this